projects/gloo/cli/pkg/cmd/license/validate_test.go

```diff
@@ -0,0 +1,60 @@
+package license
+
+import (
+	"time"
+
+	"github.com/golang-jwt/jwt/v4"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Licence", func() {
+
+	licenseClaims := LicenseClaims{
+		AddOns:         []AddOn{{Addon: 1, ExpiresAt: time.Now().Add(24 * time.Hour).Unix(), LicenseType: "trial"}},
+		ExpirationDate: time.Now().Add(24 * time.Hour).Unix(),
+		CreationDate:   time.Now().Add(-24 * time.Hour).Unix(),
+		LicenseType:    "trial",
+		Product:        "gloo",
+	}
+
+	expiredLicenseClaims := LicenseClaims{
+		AddOns:         []AddOn{{Addon: 1, ExpiresAt: time.Now().Add(24 * time.Hour).Unix(), LicenseType: "trial"}},
+		ExpirationDate: time.Now().Add(-24 * time.Hour).Unix(),
+		CreationDate:   time.Now().Add(-48 * time.Hour).Unix(),
+		LicenseType:    "ent",
+		Product:        "gloo",
+	}
+
+	token := jwt.NewWithClaims(jwt.SigningMethodHS256, licenseClaims)
+	tokenString, err := token.SignedString([]byte("secret"))
+
+	It("should verify a valid license", func() {
+		err = checkLicense(tokenString)
+		Expect(err).ToNot(HaveOccurred())
+	})
+
+	It("should fail to verify an invalid license", func() {
+		invalidLicenseKey := "invalid"
+
+		// Call the checkLicense function
+		err := checkLicense(invalidLicenseKey)
+		Expect(err).To(HaveOccurred())
+		Expect(err.Error()).To(Equal("can't parse licence key"))
```

Comment:
```suggestion
		Expect(err.Error()).To(Equal("can't parse license key"))
```
nit: consistent spelling